qipr_approver/approver/templates/templatetags/form_input.html

```diff
@@ -1,4 +1,4 @@
 <div class="form-input-wrapper input_{{ name }} ">
     <label for="{{ name }}">{{ label }}</label>
-    <input class="{{ input_class_list }}" name="{{ name }}" type="{{ type }}" value="{{ value }}" {{ is_disabled }}/>
+    <input class="{{ input_class_list }}" name="{{ name }}" type="{{ type }}" value="{{ value }}" placeholder="{{ placeholder }}" {{ is_disabled }}/>
```

Comment:
Oops sorry, I was looking at an old version. Yes, the placeholder string seems to work when there is no matching dictionary key.

On the other pull request the readonly change was an early effort. Amber liked the disabled look more anyway (its format matched our other non editable fields) so the readonly idea was dropped. Just out of curiosity, how can they change the readonly field? Could they go into the page source, alter it, and then submit?